app/assets/javascripts/components/LayerSwitcher.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+class LayerSwitcher extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    let switchers = [];
+
+    this.props.layersList.map( (layer, i) => {
+      const slug = layer.slug
+
+      switchers.push ( <div className="switcher" key={i}>
+        <input type="checkbox" id={slug}  checked={ layer.active } onChange={ () => {this.props.toggleLayers( {slug} )} }/>
```

Comment:
Better to have active class background colour for the switcher as per the designs?